### ISSUE TITLE
Enabled duration is ignored

### ISSUE THREAD

**Issue by waltersobchak40** [kbrashears5/homebridge-schedule#205]
I have version 1.0.136 installed and notice:
1.  Homebridge 1.6's plugin UI does not display "enabled duration."
2. Editing the config json to include enabledDuration does not appear to be working or respected. The switch still turns off after the default 1 second. 

Thanks!

**Comment by ghost** [comment]
#### AUTOMATICALLY GENERATED by kbrashears5/github-action-issue-notifier

Mentioning @kbrashears5 to notify them. This will be looked at soon!

**Comment by fate8383** [comment]
yeah the same issue

**Comment by Dennis-Otto** [comment]
any ideas why the config enabledDuration is not taken?

**Comment by Leftibus** [comment]
Me as well.  This schedule used to work perfectly,  now it seems maybe HomeKit architecture is not as sensitive to a quick on and off?  I just tested that the schedule is firing the switch.  Toggles at the correct time, but it is very brief, maybe 1 second.

As others have mentioned, the enable duration is not being taken into account.  Until this is fixed it seems my automations are completely broken

**Comment by yangminah** [comment]
Same issue for me.

**Comment by louisgrasset** [comment]
May it be simply because build & release CI jobs have failed?
See more details here: https://github.com/kbrashears5/typescript-homebridge-schedule/actions/runs/3982250624/jobs/6826548532

Any way to run them again just to see @kbrashears5?

**Comment by AlbertoAIG** [comment]
this plugin solve a big issue that I had with my HomeKit installation, please fix duration bug and this plugin going to be perfect @kbrashears5 

Thanks

**Comment by marzvrover** [comment]
Yeah, this just needs republished/released. I manually installed with the latest commit with the following steps:

1. `git clone http://github.com/kbrashears5/typescript-homebridge-schedule.git`
2. `cd typescript-homebridge-schedule`
3. `npm run build`
4. `npm link`

**Comment by boarochristian** [comment]
Hi everyone, has anyone managed to get the feature to work"Enable duration”?
